docs-ref-services/latest/communication-sms-readme.md

```diff
@@ -198,12 +197,12 @@ try {
 [coc]: https://opensource.microsoft.com/codeofconduct/
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
-[product_docs]: https://docs.microsoft.com/azure/communication-services/
+[product_docs]: /azure/communication-services/
```

Comment:
Do these need to be in `[link_text](link_path)` format? Does it affect the output?